core/src/main/kotlin/org/evomaster/core/problem/graphQl/resource/QuerryTypeGlobalPetsClinic.json

```diff
@@ -0,0 +1,2581 @@
+{
```

Comment:
`json` files should not be under `src/main/kotlin`, as that is only for`.kt` files.
Resource files like JSON should be either under `src/main/resources`, or under `src/test/resources`. In this case, it would be this latter

Reply:
done